library/IvozProvider/Mapper/Sql/Brands.php

```diff
@@ -62,6 +65,26 @@ protected function _save(\IvozProvider\Model\Raw\Brands $model,
         return $pk;
     }
 
+    protected function _createDomainAttrs($model)
+    {
+        $domainName = $model->getDomainUsers();
+
+        if (!empty($domainName)) {
+            $domainAttrsMapper = new \IvozProvider\Mapper\Sql\KamUsersDomainAttrs();
+
+            $domainsAttr = $domainAttrsMapper->fetchList("did='$domainName' AND name='brandId'");
+            if (empty($domainsAttr)) {
+                $domainAttr = new \IvozProvider\Model\KamUsersDomainAttrs();
+
+                $domainAttr->setDid($domainName)
+                           ->setName('brandId')
+                           ->setType('0')
```

Comment:
What does 0 stand for? Could we add some constants within Model\KamUsersDomainAttrs in order to make this easier to understand?